packages/kamino-sdk/src/utils/utils.ts

```diff
@@ -98,3 +98,13 @@ export async function getUpdateStrategyConfigIx(
 
   return updateStrategyConfig(args, accounts);
 }
+
+export function collToLamportsDecimal(amount: Decimal, decimals: number): Decimal {
+  let factor = Math.pow(10, decimals);
```

Comment:
I think you should be using decimaljs method to avoid situations where the number is too big, Decimal.pow I think is the method